build/local_install.R

```diff
@@ -8,14 +8,14 @@ local({r <- getOption("repos")
 
 library(devtools)
 
-install.packages(c("covidcast","data.table","vroom","dplyr","RSocrata"), force=TRUE)
+install.packages(c("covidcast","data.table","vroom","dplyr","RSocrata"), quiet=TRUE)
 # devtools::install_github("hrbrmstr/cdcfluview")
 
 # To run if operating in the container
 initial.options <- commandArgs(trailingOnly = FALSE)
 file.arg.name <- "--file="
 script.name <- sub(file.arg.name, "", initial.options[grep(file.arg.name, initial.options)]) # get the name of this file, by looking for the option "--file" in the arguments that were used to start this R instance and getting the term that comes after
-pkg.dir <- paste0(dirname(script.name), "../R_packages/") # find the directory that this file is within
+pkg.dir <- paste0(dirname(script.name), "/../flepimop/R_packages/") # find the directory that this file is within
```

Comment:
Interesting. This work on conda but it would be necessary to have it work on docker as well. I don't fully understand how it can fail on docker though. This script is in `build/`, so `dirname(script.name)` returns `$FLEPI_PATH/build`. The packages are in `$FLEPI_PATH/flepimop/R_packages` so I feel like `/../flepimop/R_packages/`should redirect to the right place, no?

Reply:
In the current Dockerfile, "COPY flepimop/R_packages $HOME/R_packages" is described compliant with the former csp information. So to run the modification part, some are needed as well as _dockerbuild, but now it is okay as noted above.

Reply:
Needed how to reinstall packages from git (between different repos) though, in this case we will modify Dockerfifile to change COPY source and destination dir, not to remain the former dir information in the docker image. I think it may work (and I will test it)